dmutils/content_loader.py

```diff
@@ -74,6 +103,115 @@ def _question_should_be_shown(self, dependencies, service_data):
                 return False
         return True
 
+    def get_question(self, question_id):
+        for section in self.sections:
+            question = section.get_question(question_id)
+            if question:
+                return question
+
+
+class ContentSection(object):
+    @classmethod
+    def create(cls, section):
+        if isinstance(section, ContentSection):
+            return section
```

Comment:
This should make a copy as section `.questions` might be modified by additional filters. I think we can rely on `__getitem__` for now to reuse the code from the `else:` branch